src/plugins/filters/components/FilterObject.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+<div>
+    <div class="c-tree__item menus-to-left"
+        @click="toggleExpanded">
+        <span class="c-disclosure-triangle is-enabled flex-elem"
+            :class="{'c-disclosure-triangle--expanded': expanded}"></span>
+        <div class="c-tree__item__label">
+            <div class="t-object-label l-flex-row flex-elem grows">
+                <div class="t-item-icon flex-elem"
+                    :class="objectCssClass">
+                </div>
+                <div class="t-title-label flex-elem grows">{{ filterObject.name }}</div>
+            </div>
+        </div>
+    </div>
+    <ul class="c-properties__section" v-if="expanded">
+        <filter-field
+            v-for="field in filterObject.valuesWithFilters"
+            :key="field.key"
+            :filterField="field"
+            :persistedFilters="persistedFilters[field.key]"
+            @onUserSelect="collectUserSelects"
+            @onTextEnter="updateTextFilter">
+        </filter-field>
+    </ul>
+</div>
+</template>
+
+<style lang="scss">
+
+</style>
+
+<script>
+import FilterField from './FilterField.vue';
+
+export default {
+    inject: ['openmct'],
+    components: {
+        FilterField
+    },
+    props: {
+        filterObject: Object, 
+        persistedFilters: {
+            type: Object,
+            default: () => {
+                return {};
+            }
+        }
+    },
+    data() {
+        return {
+            expanded: false,
+            objectCssClass: undefined,
+            updatedFilters: this.persistedFilters
+        }
+    },
+    methods: {
+        toggleExpanded() {
+            this.expanded = !this.expanded;
+        },
+        collectUserSelects(key, comparator, valueName, value) {
+            let filterValue = this.updatedFilters[key];
+
+            if (filterValue && filterValue.comparator === comparator) {
+                if (value === false) {
+                    filterValue.values = filterValue.values.filter(v => v !== valueName);
+                } else {
+                    filterValue.values.push(valueName);
+                }
+            } else {
+                this.updatedFilters[key] = {
```

Comment:
On second thought, the `comparator` form is really just to support `possibleValues`. Many filters will be defined in the metadata like:
```Javascript
filters: ['equals', 'greaterThan', 'lessThan']
```
I think that irrespective of how they are defined in the metadata, we should always provide the filters to telemetry providers in a simple and succinct form such as -

``` JavaScript
domainObject.filters = [
    'dss_id': {
        'equals': []
    },
    'alarm_level': {
        'greaterThan': 1
    }
]
```
Ie. without the 'comparator'

Reply:
This could be done in the Inspector, and still passed straight through to the Telemetry API from the views.

Reply:
I made the changes with one difference  - 
`
domainObject.filters = {
    telemetryId#1: {
          'dss_id': {
              'equals': []
           },
          'alarm_level': {
              'greaterThan': 1
           }
     }
}
`

Reply:
Ah, yep, good catch.